Changes to be committed:

NevasPrime opened the pull request:
new file:   nplbam/app/jsons/dog_questions.txt
	- Added a json of all the questions related to dogs
	modified:   nplbam/app/new_animal.py
        - Changed to get json of dog questions
        - Changed to new template for all animals, not just dog
        - Need to still hook up to database
	new file:   nplbam/app/templates/add_animal.html
        - New Template which replaces Add_Dog
        - Uses Jinga2 templating to dynamically change based on animal
        - Still needs to be hooked up for submitting files/pictures
	deleted:    nplbam/app/templates/add_dog.html
        - See Above
	modified:   nplbam/app/templates/animals.html
        - Now uses Jinja2 Templating to show the navigation bar
	new file:   nplbam/app/templates/base_template.html
        - New Base templating from which all but index will inherit from.
        - Has the nav bar and head. Still needs to do roles
	deleted:    nplbam/app/templates/header.html
        - Useless, replaced by Above
	modified:   start_flask.sh
        - Can now start interactive mode using -i. YaaaaaaY

NevasPrime committed:
Changes to be committed:
	new file:   nplbam/app/jsons/dog_questions.txt
	- Added a json of all the questions related to dogs
	modified:   nplbam/app/new_animal.py
        - Changed to get json of dog questions
        - Changed to new template for all animals, not just dog
        - Need to still hook up to database
	new file:   nplbam/app/templates/add_animal.html
        - New Template which replaces Add_Dog
        - Uses Jinga2 templating to dynamically change based on animal
        - Still needs to be hooked up for submitting files/pictures
	deleted:    nplbam/app/templates/add_dog.html
        - See Above
	modified:   nplbam/app/templates/animals.html
        - Now uses Jinja2 Templating to show the navigation bar
	new file:   nplbam/app/templates/base_template.html
        - New Base templating from which all but index will inherit from.
        - Has the nav bar and head. Still needs to do roles
	deleted:    nplbam/app/templates/header.html
        - Useless, replaced by Above
	modified:   start_flask.sh
        - Can now start interactive mode using -i. YaaaaaaY

NevasPrime committed:
Changes to be committed:
	new file:   nplbam/app/jsons/dog_questions.json
         - Changed to a json.
         - Used online tool to format the json into more human readable
	deleted:    nplbam/app/jsons/dog_questions.txt
         - As Above
	modified:   nplbam/app/new_animal.py
         - changed comment to """ for documentation

ValliereMagic commented on nplbam/app/new_animal.py:
```diff
@@ -1,10 +1,24 @@
 from flask import render_template, Blueprint, session as flask_session, redirect
```
Note to self for later: Go through and add docstrings to pretty much every function.

ValliereMagic reviewed (changes requested):
All good to go once the json generation file is added.

NevasPrime committed:
Changes to be committed:
	new file:   nplbam/app/jsons/json_creator/dogQ.py
        - This file was used to create the json. Just adding it to the work.
